modules/shared/src/main/scala/org/tessellation/shared/package.scala

```diff
@@ -44,7 +45,7 @@ package object shared {
     classOf[CommonRumorBinary] -> 323,
     classOf[Transaction] -> 324,
     classOf[TransactionReference] -> 325,
-    classOf[SubHeight] -> 326
```

Comment:
why it's removed? 🤔

Reply:
It's a `@newtype` now (`Refine` is registered).